transforms/angle-brackets/index.js

```diff
@@ -27,8 +28,9 @@ function getOptions() {
 }
 
 module.exports = function(file) {
+  let helpers = getHelperData(getTelemetry());
```

Comment:
FWIW, I think we also want to know the names of the components too, don't we? Seems fine for a follow up, but I do think there are scenarios that we can make better with this.

For example, with the following template when `{{foo}}` is not a helper how do we know if it is a local property (and should be left alone) or a component invocation (which should be transformed)?

Reply:
This PR know enumerates components

Reply:
awesome